### PR TITLE
add a real grid scan

### PR DESCRIPTION
**Pull request by stufisher**

### CODE REVIEW

**Comment by stufisher** [comment]
wait to tests to make sure i havnt screwed up the fks

File: schemas/ispyb/data.sql [hunk 1]
```diff
@@ -617,6 +619,7 @@ INSERT INTO `ExperimentType` (`experimentTypeId`, `name`, `proposalType`, `activ
 --
 
 /*!40000 ALTER TABLE `GridInfo` DISABLE KEYS */;
+INSERT INTO `GridInfo` (`gridInfoId`, `dataCollectionGroupId`, `xOffset`, `yOffset`, `dx_mm`, `dy_mm`, `steps_x`, `steps_y`, `meshAngle`, `pixelsPerMicronX`, `pixelsPerMicronY`, `snapshot_offsetXPixel`, `snapshot_offsetYPixel`, `recordTimeStamp`, `orientation`, `workflowMeshId`, `snaked`, `dataCollectionId`) VALUES (1281212,5440739,NULL,NULL,0.02,0.02,30,16,NULL,0.83,0.83,304,229.75,'2021-02-25 10:15:06','horizontal',NULL,1,NULL);
```
Comment (Anthchirp) [inline schemas/ispyb/data.sql:622]: Just wondering - the `dataCollectionId` in the GridInfo record is empty on purpose?

**Review by Anthchirp — APPROVED** [review]
Looking good, thank you!

File: schemas/ispyb/data.sql (hunk 1 above)
Comment (stufisher) [reply schemas/ispyb/data.sql:622]: Dont think gda is populating dc yet so its still using the old dcg linkage. I can add test data to the relevant dc if thats useful?

Comment (stufisher) [reply schemas/ispyb/data.sql]: Added

**Review by KarlLevik — APPROVED** [review]

**Pull request merged**